### PR TITLE
scala 3.2.0, scala@2.13 2.13.8 (new formula)

### PR DESCRIPTION
**Pull request by cho-m**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/scala.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
   homepage "https://www.scala-lang.org/"
-  url "https://downloads.lightbend.com/scala/2.13.8/scala-2.13.8.tgz"
-  mirror "https://www.scala-lang.org/files/archive/scala-2.13.8.tgz"
-  mirror "https://downloads.typesafe.com/scala/2.13.8/scala-2.13.8.tgz"
-  sha256 "2cb31d8469c651839f0e9c837a1ab06550d031726752f54906be1b9de01314cf"
+  url "https://github.com/lampepfl/dotty/releases/download/3.1.3/scala3-3.1.3.tar.gz"
+  sha256 "9e1eefdcab77b2d2a9057b3d6f78301591e9c27513c92413f3c353a77093f2d7"
   license "Apache-2.0"
 
   livecheck do
```
Comment (SMillerDev) [inline Formula/scala.rb:8]: We can check https://www.scala-lang.org/download/ for the latest version, or use github releases

**Comment by armanbilge** [comment]
Scala 3.2.0 is out now: https://www.scala-lang.org/blog/2022/09/05/scala-3.2.0-released.html

**Review by chenrui333 — COMMENTED** [review]
rather than using `scala@3.2` as alias, maybe `scala@3` is better?

**Comment by armanbilge** [comment]
`scala@3` is fine. Note that unlike Scala 2, Scala 3 is is fully backwards compatible.
- programs compiled with Scala 2.12 only run on Scala 2.12
- programs compiled with Scala 2.13 only run on Scala 2.13
- programs compiled with Scala 3.x run on any Scala 3.y where y >= x. For example, Scala 3.2.0 can run programs compiled with Scala 3.0.0 and 3.1.0 too.

**Comment by chenrui333** [comment]
> `scala@3` is fine. Note that unlike Scala 2, Scala 3 is is fully backwards compatible.
> 
> * programs compiled with Scala 2.12 only run on Scala 2.12
> * programs compiled with Scala 2.13 only run on Scala 2.13
> * programs compiled with Scala 3.x run on any Scala 3.y where y >= x. For example, Scala 3.2.0 can run programs compiled with Scala 3.0.0 and 3.1.0 too.

that is exactly why I think `scala@3` is more appropriate. :)

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**